prime-router/settings/staging/0149-etor.yml

```diff
@@ -86,6 +86,7 @@
       transport: !<REST>
         reportUrl: "https://cdcti-staging-api.azurewebsites.net/v1/etor/demographics"
```

Comment:
I wonder if we need to create another receiver for the `v1/etor/orders` endpoint. Something along the lines of: `etor-service-receiver-demographics` and `etor-service-receiver-orders` , to tell the difference. Thoughts?

Reply:
Is there any field in the resource bundle that can be used to distinguish between the two?  If so, we don't need to create any additional endpoint.

Reply:
Good thought.  Yes, there should be a field that we can differentiate.

In the FHIR that goes out, there will be a `MessageHeader` resource that has a `event` (`eventCoding`) with a `code` of `O01`.  That means our `orders` endpoint.

If the `MessageHeader` resource has the `code` of `A01`, that means our `demographics` endpoint.

Reply:
In the bundle there is resource.activity.coding.code to identify as "Laboratory order" or "result".  Can you use that field to distinguish resource type?

Reply:
Are you talking about the `Provenance` resource?  If so, I tentatively say yes, that will be either `O01` or `A01` too.  I say "tentatively" because that violates the FHIR spec for that field.

Reply:
Yes, I was talking about the Provenance resource.  Would you let us know why it violates the FHIR spec?  Please give further info.

Reply:
I don't want to complicate this PR too much, so I think this will work for now.

Specifically, the FHIR spec mentions there are only specific values/system allowed for `activity`.  https://www.hl7.org/fhir/R4/valueset-provenance-activity-type.html